src/contrib/mlir/compiler/compiler.cpp

```diff
@@ -629,9 +629,22 @@ mlir::Operation* MLIRCompiler::create_generic_op(const ngraph::Node* ng_node)
 {
     std::vector<mlir::Value*> arg_values;
     std::vector<mlir::Type> res_types;
+    auto input_map = m_compiled_kernel->get_input_map();
+    std::shared_ptr<descriptor::Tensor> arg_tensor;
     for (auto& arg : ng_node->get_arguments())
```

Comment:
If you switched this to use `input_values` it would work with values from multi-output nodes.